src/main/java/com/github/fishio/PlayerFish.java

```diff
@@ -147,7 +149,11 @@ public void render(GraphicsContext gc) {
 		}
 		
 		getBoundingArea().setRotation(behaviour); //update rotation;
-		drawRotatedImage(gc, sprite, getBoundingArea());
+		
+		//Only render if the fish has a sprite
+		if (sprite != null) {
+			drawRotatedImage(gc, sprite, getBoundingArea());
+		}
```

Comment:
you could add an else case here that calls the render method of the superClass (Entity), so you at least something is rendered.

<organisation